schedule/yam/agama/agama_install_with_cryptfs.yaml

```diff
@@ -0,0 +1,18 @@
+---
+name: agama_installation
+description: >
+  Playwright test on agamaq
+vars:
+  ENCRYPT: 1
+  DESKTOP: textmode
+schedule:
+  - installation/bootloader_start
+  - yam/agama/patch_agama
+  - yam/agama/agama_install_with_cryptfs
```

Comment:
`agama_full_disk_encryption` should be more readable than referring to internals like cryptfs...

Reply:
Done.